modules/crawler.py

```diff
@@ -201,7 +201,7 @@ async def sitemap():
 			links = sm_soup.find_all('loc')
 			for url in links:
 				url = url.get_text()
-				if url != None:
+				if url is not None:
```

Comment:
When I switched to python 3.8 I got lot of warnings for this and as per python 3.8 changelog : 
```
The compiler now produces a SyntaxWarning when identity checks (is and is not) are used with certain types of literals (e.g. strings, numbers). These can often work by accident in CPython, but are not guaranteed by the language spec. The warning advises users to use equality tests (== and !=) instead. 
```